docs/data/data-grid/events/CatalogOfEventsNoSnap.js

```diff
@@ -42,16 +42,16 @@ const EventRow = ({ event }) => {
           .join(', ')})
 <DataGrid
   ${event.componentProp}={handleEvent}
-  {...other} 
+  {...other}
 />`
       : '';
 
     return `
 const handleEvent: GridEventListener<'${event.name}'> = (
   ${args.join('\n  ')}
-) => {...}    
-  
-// Imperative subscription    
+) => {...}
```

Comment:
Something I found by chance, it's not ideal to have these trailing spaces when you use https://next.mui.com/x/react-data-grid/events/#catalog-of-events

<img width="602" alt="Screenshot 2022-10-08 at 22 08 42" src="https://user-images.githubusercontent.com/3165635/194726129-7e22c4ed-b1e5-4751-b2c3-fab63d6287f6.png">

Reply:
I think that we could enforce a rule in the CI with https://github.com/mui/material-ui/pull/34686#discussion_r991169741 to have the PRs fail if they have trailing spaces.